MaterialComponents.podspec

```diff
@@ -250,6 +250,16 @@ Pod::Spec.new do |s|
     end
   end
 
+  s.subspec "MaskedTransition" do |ss|
+    ss.ios.deployment_target = '8.0'
+    ss.public_header_files = "components/#{ss.base_name}/src/*.h"
+    ss.source_files = "components/#{ss.base_name}/src/*.{h,m}", "components/#{ss.base_name}/src/private/*.{h,m}"
+
+    ss.dependency "Transitioning", "~> 1.1"
```

Comment:
Transitioning isn't an MDC library - is its own CocoaPod and is already published to CocoaPods.